evm/15_core-concepts/46_EOAs-vs-Smart-Contracts.md

```diff
@@ -0,0 +1,85 @@
+---
+title: EOAs vs Smart Contracts
+contributors:
+  - { name: Esteban Saá, github: stebansaa }
+---
+  
+## EOAs vs Smart Contracts
```

Comment:
You don't need to add this here, the `title` in the frontmatter above will already show up. This would duplicate that title

Reply:
ok.